docs/_static/theme_override.css

```diff
@@ -6,5 +18,4 @@
 
 .wy-nav-top {
     background: #009688;   
-}
-
+}
```

Comment:
Note that the last _character_ in a file should be a newline. You can check the "Files Changed" tab to see if you've done this correctly -- it will show a red circle if not.

(The original comment was to remove the additional blank line, but this line should still be newline-terminated.)

Reply:
Thanks for the explanation, I hope I've got it right. Done.